src/top/metrics.rs

```diff
@@ -154,12 +193,14 @@ pub fn subscribe(client: SubscriptionClient, tx: state::EventTx, interval: i64)
 
     tokio::spawn(component_added(Arc::clone(&client), tx.clone()));
     tokio::spawn(component_removed(Arc::clone(&client), tx.clone()));
-    tokio::spawn(processed_events_totals(
+    tokio::spawn(events_in_total(Arc::clone(&client), tx.clone(), interval));
```

Comment:
```suggestion
    tokio::spawn(events_in_totals(Arc::clone(&client), tx.clone(), interval));
```

To match the previous suggestion.